packages/firestore/test/unit/local/persistence_test_helpers.ts

```diff
@@ -44,3 +55,42 @@ export async function testMemoryPersistence(): Promise<MemoryPersistence> {
   await persistence.start();
   return persistence;
 }
+
+/**
+ * Creates LocalStorageNotificationChannel instance for testing, destroying any
+ * previous contents if they existed.
+ */
+export async function testLocalStorageNotificationChannel(
+  ownerId: string,
+  existingMutationBatches: BatchId[],
+  existingQueryTargets: TargetId[]
```

Comment:
BatchIds, TargetIds

Reply:
Updated, but I don't think this is more of a nice-to-have since this is typed to begin with.

Reply:
Yeah.  But since we also have actual MutationBatch and Target classes in the client, it's confusing to mismatch the names and types (for humans, that is...  the compiler can keep it straight :-))